treeherder/model/tasks.py

```diff
@@ -90,44 +90,44 @@ def publish_job_action(project, action, job_id, requester):
     )
 
 
-@task(name='publish-resultset-action')
-def publish_resultset_action(project, action, resultset_id, requester, times=1):
+@task(name='publish-push-action')
+def publish_push_action(project, action, push_id, requester, times=1):
     newrelic.agent.add_custom_parameter("project", project)
     newrelic.agent.add_custom_parameter("action", action)
-    newrelic.agent.add_custom_parameter("resultset_id", str(resultset_id))
+    newrelic.agent.add_custom_parameter("push_id", str(push_id))
     newrelic.agent.add_custom_parameter("requester", requester)
 
     publisher = pulse_connection.get_publisher()
     if not publisher:
         return
 
-    publisher.resultset_action(
+    publisher.push_action(
         version=1,
         project=project,
         action=action,
         requester=requester,
-        resultset_id=resultset_id,
+        resultset_id=push_id,
+        push_id=push_id,
         times=times
     )
 
 
-@task(name='publish-resultset-runnable-job-action')
-def publish_resultset_runnable_job_action(project, resultset_id, requester,
-                                          requested_jobs, decision_task_id):
+@task(name='publish-push-runnable-job-action')
+def publish_push_runnable_job_action(project, push_id, requester, requested_jobs, decision_task_id):
```

Comment:
The way these async celery tasks work, is that one part of Treeherder schedules a named task, it then gets added to the queue on the Rabbitmq server, and then a worker later fetches the task from the queue and tries to match the name of the task against those defined. In practice that means that a race condition can occur, whereby a task gets queued up with the old name, but doesn't get run before the deployment of this PR completes, and so the worker then receives a task that doesn't match any known task, since they now have the new names.

As such I think we'll need to keep both the old and new task name here for now, so there aren't tasks silently dropped when this is deployed. eg: 
* Add the new task name alongside the old task name
* Make the parts of Treeherder that schedule the task use the new name
* Deploy those changes
* Later on in another deployment, remove the old task name since the queues will now definitely be empty.